geomet/wkt.py

```diff
@@ -509,8 +509,6 @@ def _load_multipoint(tokens, string):
                 paren_depth -= 1
                 if paren_depth == 0:
                     break
-            elif t == '':
```

Comment:
So, I removed this `elif` clause because I couldn't find any test case which covered it. The only way we can get here is if we take the last item from the generator; the token which looks like:

```python
TokenInfo(type=0 (ENDMARKER), string='', start=(2, 0), end=(2, 0), line='')
```
I think that can only happen if we don't hit the `break` above, which in turn will only happen if we have misbalanced parens, like one missing off the end. However, in that case, `tokenize.generate_tokens` will always throw an error, which is being caught and handled in this `try`. I added some tests cases in this PR which prove as much.

Reply:
Awesome, thanks for cleaning that up!